src/components/edit-project/editProject.js

```diff
@@ -63,20 +68,25 @@ angular.module("openshiftCommonUI")
                 cleanEditableAnnotations(mergeEditable($scope.project, $scope.editableFields)),
                 {projectName: $scope.project.name},
                 {errorNotification: false})
-              .then(function() {
+              .then(function(project) {
                 // angular is actually wrapping the redirect action :/
                 var cb = $scope.redirectAction();
                 if (cb) {
                   cb(encodeURIComponent($scope.project.metadata.name));
                 }
+
+                showAlert({
+                  type: "success",
+                  message: "Project \'"  + displayNameFilter(project) + "\' was successfully modified."
```

Comment:
nit: I think "was successfully updated." is slightly better and consistent with our other messages